src/app/ui/AppProvider/ToastProvider.tsx

```diff
@@ -27,15 +39,70 @@ const Toast: FC<ToastProps> = ({ message }) => (
   </Box>
 );
 
-const ToastProvider: FC<PropsWithChildren> = ({ children }) => (
-  <RNTNToastProvider
-    offsetBottom={20}
-    renderToast={({ message }) => {
-      return <Toast message={message} />;
-    }}
-  >
-    {children}
-  </RNTNToastProvider>
-);
+const getDefaultBannerProps = (props: ToastProps) => {
+  let type = props.type?.replace('banner', '').toLowerCase(),
+    icon = props.icon,
+    bgColor = '';
+
+  switch (type) {
+    case 'success':
+      icon = <OcticonsCircleCheckFill color={colors.alertSuccess} size={24} />;
+      bgColor = colors.alertSuccessBg;
+      break;
+    case 'danger':
+      icon = <MaterialAlertOctagon color={colors.alertError} size={26} />;
+      bgColor = colors.alertErrorBg;
+      break;
+    default:
+      break;
+  }
+
+  return { icon, type, bgColor };
+};
+
+const Banner: FC<ToastProps> = (props: ToastProps) => {
+  const { top } = useSafeAreaInsets();
+  const { icon, bgColor } = getDefaultBannerProps(props);
+  return (
+    <Box
+      backgroundColor={bgColor}
+      width="100%"
+      py={40}
```

Comment:
Should be a smaller value. See the corresponding [banner design](https://www.figma.com/file/vFUwmYD8Ya0ShZsJJ2f757/Web-app?type=design&node-id=1535-31902&mode=design) for the web app, which Mike X mentioned we can use as reference for mobile